src/js/tracks/text-track-display.js

```diff
@@ -92,6 +92,7 @@ class TextTrackDisplay extends Component {
 
     player.on('loadstart', Fn.bind(this, this.toggleDisplay));
     player.on('texttrackchange', Fn.bind(this, this.updateDisplay));
+    player.on('loadstart', Fn.bind(this, this.preselectTrack));
```

Comment:
I believe that we discussed it and decided that it would be disorienting to preselect a new track if a track had already been selected by the user.